torch/distributed/_shard/sharding_spec/chunk_sharding_spec_ops/linear.py

```diff
@@ -357,8 +353,9 @@ def forward(ctx, world_size, start_pos, chunk_size, weight, pg, bias):
     @staticmethod
     def backward(ctx, grad_output):
         results = []
+        grad_output_cloned = grad_output.clone()
         for idx in range(ctx.world_size):
-            results.append(grad_output.clone())
+            results.append(grad_output_cloned)
```

Comment:
Could this be reformulated as `results = [grad_output.clone()] * ctx.world_size`?